### PR TITLE
Show symbol actions by client capabilities

### PR DESCRIPTION
**Pull request by ayoub-benali**
Show only the actions that the server can handle during the hover event

### CODE REVIEW

**Comment by tomv564** [comment]
Can you please put in the effort and write a description for your pull request? For example, I guess this is for issue #482 ?

File: plugin/hover.py [hunk 1]
```diff
@@ -82,18 +82,25 @@ def handle_response(self, response: 'Optional[Any]', point) -> None:
...
-    def symbol_actions_content(self):
+    def symbol_actions_content(self, point):
         actions = []
-        # TODO: filter by client capabilities
-        actions.append("<a href='{}'>{}</a>".format('definition', 'Definition'))
-        actions.append("<a href='{}'>{}</a>".format('references', 'References'))
-        actions.append("<a href='{}'>{}</a>".format('rename', 'Rename'))
+        (x, y) = self.view.rowcol(point)
+        event = {"x": x, "y": y}
+        if self.has_client_with_capability('definitionProvider') and is_at_word(self.view, event):
```
Comment (tomv564) [inline plugin/hover.py]: I would suggest not including `is_at_word` in every statement but just doing it once. 

But even more so, this check is already done before the hover request is made (see `is_likely_at_symbol`) so perhaps we can avoid doing the same check a different way again?

**Comment by ayoub-benali** [comment]
> Can you please put in the effort and write a description for your pull request? For example, I guess this is for issue #482 ?

The PR is simply taking care of a todo left in the code. Regarding https://github.com/tomv564/LSP/issues/482 I can add flag if needed

File: plugin/hover.py (hunk 1 above)
Comment (ayoub-benali) [reply plugin/hover.py]: @tomv564  so you are saying `is_at_word` check can be ignore and the client capability check is enough ?

**Comment by ayoub-benali** [comment]
@tomv564 are you happy with the changes ?

File: plugin/hover.py (hunk 1 above)
Comment (tomv564) [reply plugin/hover.py]: Changes are good to the first part of my comment!

But did you see that we already check if the cursor is at a symbol? Is there a need to check it again here?

Comment (ayoub-benali) [reply plugin/hover.py]: @tomv564 you are right, no need to check

**Comment by tomv564** [comment]
Thanks, great work!

**Pull request merged**